spec/integration/dataset_test.rb

```diff
@@ -108,7 +108,7 @@
     @ds.all.must_equal [{:id=>1, :number=>11}]
   end
   
-  cspecify "should have update return the number of matched rows", [:do, :mysql], [:ado] do
+  cspecify "should have update return the number of matched rows", [:do, :mysql] do
     @ds.update(:number=>:number).must_equal 1
```

Comment:
This is the only `skip` that no longer needs to be. All other `skips` are still legit as far as I can tell.